wordpress-baseline/docker-entrypoint-baseline.sh

```diff
@@ -72,9 +72,34 @@ uninitialized_baseline() {
   [ -n "$(grep 'localhost' $WORDPRESS_CONF)" ] && true || false
 }
 
-# No shib key or cert? Then no shibboleth sp configuration.
-requireShibboleth() {
-  ([ -n "$SHIB_SP_KEY" ] && [ -n "$SHIB_SP_CERT" ]) && true || false
+MU_PLUGIN_LOADER='/var/www/html/wp-content/mu-plugins/loader.php'
+check_mu_plugin_loader() {
+  if [ -f $MU_PLUGIN_LOADER ] ; then
+    echo "mu_plugin_loader already generated..."
+  else
+    echo "generate_mu_plugin_loader..."
+    wp bu-core generate-mu-plugin-loader \
+      --path=/var/www/html \
+      --require=/var/www/html/wp-content/mu-plugins/bu-core/src/wp-cli.php
+  fi
+}
+
+MULTISITE_LOG='/var/www/html/multisite.log'
```

Comment:
Wouldn't this file be publicly available? Do we really want that?

Reply:
This file is created as an extra bash command executed right after calling wp core multisite-install:
date > $MULTISITE_LOG
It simply serves to indicate that the command was run once.
If there is a cleaner way to do this, like looking for some file that multisite-install generates itself, I'm all ears.

Reply:
This is okay for now, we can continue to evolve this feature as we continue working with the container.